build.gradle

```diff
@@ -117,6 +117,38 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "hirakiuc"
+                        email = "hirakiuc@gmail.com"
+                    }
+                    developer {
+                        name = "Toyama Hiroshi"
+                        email = "toyama0919@gmail.com"
+                    }
```

Comment:
@toyama0919 Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-elasticsearch`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-elasticsearch` on Maven Central, from your work in #14. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want to be there, or if you want to be listed in another name/email.

Reply:
no problem! LGTM :+1: